homeassistant/components/control4/manifest.json

```diff
@@ -6,7 +6,7 @@
   "documentation": "https://www.home-assistant.io/integrations/control4",
   "iot_class": "local_polling",
   "loggers": ["pyControl4"],
-  "requirements": ["pyControl4==0.0.6"],
+  "requirements": ["pyControl4==1.1.0"],
```

Comment:
I suggest that you move this to a preliminary PR.

Reply:
got it

Reply:
https://github.com/home-assistant/core/pull/90115